common/repo/tests/GitRepoTest.java

```diff
@@ -156,5 +161,149 @@ public void testGetMultipleCommits() {
                 .build()),
         repo.getCommits(TEST_BRANCH));
   }
+
+  @Test
+  public void testAddFile() {
```

Comment:
Looks like it's not just `add`, so how about:
s/testAddFile/testCommitFile or testAddAndCommitFile`

Reply:
Fixed.